### ISSUE TITLE
Add chartability points to our accessibility checking guidance

### ISSUE THREAD

**Issue by mzayeddfe** [dfe-analytical-services/analysts-guide#207]
**Is your feature request related to a problem? Please describe.**
Don’t need to do everything from here, but should review and add any we feel we should be checking for in dashboards:

[Workbook](https://chartability.github.io/POUR-CAF/)

**Describe the solution you'd like**
Review and add anything we feel we should be checking for in dashboards:

[Workbook](https://chartability.github.io/POUR-CAF/)
